Gemfile

```diff
@@ -32,4 +32,5 @@ group :test, :development do
   gem 'flamegraph',     '~> 0.0'
   gem 'rspec-prof',     '~> 0.0'
   gem 'byebug'
+  gem 'test-unit'
```

Comment:
Oke, wel handig om even een TODO met deze uitleg toe te voegen; ik zou zoiets in ieder geval vergeten later weg te halen.